pkg/accounts/adaptor/repository/dummy/avatar.ts

```diff
@@ -67,6 +67,21 @@ export class InMemoryAccountAvatarRepository
 
     return Result.ok(this.media.get(mediumID) as Medium);
   }
+
+  async findByIDs(
+    accountIDs: AccountID[],
```

Comment:
```suggestion
    accountIDs: readonly AccountID[],
```
TypeScript の判定が緩いのか, 互換性のない型でメソッドが実装されています. `interface` のものと同じように `readonly` を指定しましょう. `InMemoryAccountHeaderRepository` の `findByIDs` も同様です.